api/v1alpha2/firewallrule_types.go

```diff
@@ -80,6 +102,7 @@ type FirewallRule struct {
 type FirewallRuleList struct {
 	metav1.TypeMeta `json:",inline"`
 	// metadata is the standard object's metadata.
+	// +optional
```

Comment:
Missing optional below for `Items`? But also, wouldn't `Items` for all the `*List` structs be `required` since we don't have `omitempty`?